app/controllers/users_controller.rb

```diff
@@ -1,36 +1,41 @@
 class UsersController < ApplicationController
 
-  def create
+  def sign_in
     auth_hash = request.env['omniauth.auth']
     email = auth_hash['info']['email']
     user = User.find_or_create_by(email: email)
     session[:access_token] = auth_hash['info']['email']
 
     UserFacade.add_user(email)
-    if !user.dob || !user.name
-      redirect_to "/register"
+    if !user.zodiac_sign || !user.name
+      redirect_to "/sign_up"
     else
       redirect_to "/dashboard"
     end
   end
 
-  def index
-    @user = User.find_by(email: session[:access_token])
-    if !@user.dob
-      redirect_to "/register"
-    end
-    zodiac_sign = "leo"
-    day = "today"
-    @horoscope = HoroscopeFacade.get_today_horoscope(zodiac_sign, day)
-  end
+  def sign_up 
+    user = User.find_by(email: session[:access_token])
+    if !user.zodiac_sign || !user.name
```

Comment:
neither here nor there, but I love that there's an attribute called `zodiac_sign`